js/modules/k6/http/response.go

```diff
@@ -208,3 +226,25 @@ func (res *Response) ClickLink(args ...goja.Value) (*Response, error) {
 
 	return New().Get(res.GetCtx(), rt.ToValue(requestURL.String()), requestParams)
 }
+
+func checkErrorInJSON(input []byte, offset int, err error) error {
+	lf := '\n'
+	str := string(input)
+
+	// Humans tend to count from 1.
+	line := 1
+	character := 0
+
+	for i, b := range str {
+		if b == lf {
+			line++
+			character = 0
+		}
+		character++
+		if i == offset {
+			break
+		}
+	}
+
+	return fmt.Errorf("cannot parse json schema due to a error at line %d, character %d , error: %v", line, character, err)
```

Comment:
Maybe you can create a separate error like in https://github.com/loadimpact/k6/commit/8af7f5c852e7876807e95ed4ac1dc0658b047010 you can than make the `fmt.Sprintf` statement into a separate lines and this will probably reduce the size enough. And use `//nolint:lll` if that doesn't bring the char count below 120
You can than reuse the error in the tests so that the message you expect is `"GoError: " + expectedErr.Error()` again limiting the size of the of line and also helping if we ever decide to change the error message.
Also `json schema` is a particular thing that we are not parsing here we are parsing just json. Also "du to a**n** error"